src/sentry/tasks/integrations/slack/post_message.py

```diff
@@ -15,9 +17,12 @@
     max_retries=0,
 )
 def post_message(
-    payload: Mapping[str, Any], log_error_message: str, log_params: Mapping[str, Any]
+    payload: Mapping[str, Any],
+    log_error_message: str,
+    log_params: Mapping[str, Any],
+    integration_id: int | None = None,
```

Comment:
We need `integration_id` to have a default because this task is triggered in getsentry. The steps are:

1. Add the new param + default here
2. Remove direct token usage from getsentry, and provide integration_id to this task
3. Remove this default for the task